lib/Collection.js

```diff
@@ -0,0 +1,197 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const axios = require('axios');
+const bedrock = require('bedrock');
+const {EdvClient, EdvDocument} = require('edv-client');
+const https = require('https');
+const instances = require('./instances');
+const {util: {BedrockError}} = bedrock;
+
+const DEFAULT_HEADERS = {Accept: 'application/ld+json, application/json'};
+
+module.exports = class Collection {
+  constructor({type, instance, controllerKey, capability}) {
+    this.type = type;
+    this.instance = instance;
+    this.capability = capability;
+    this.controllerKey = controllerKey;
+  }
+
+  // FIXME: bikeshed name
+  static async getInstance({type, instance}) {
+    if(typeof instance === 'string') {
+      ({instance} = await instances.get({id: instance}));
+    }
+    const controllerKey = await instances.getControllerKey({id: instance.id});
+    const capability = await instances.getCapability(
+      {instance, referenceId: `${instance.id}-edv-configuration`});
```

Comment:
What's going on with the suffix `edv-configuration` here?

Reply:
It's the suffix used throughout the application that identifies the zcap associated with reading/write to the application's vault.

Reply:
Is there or would there be a collision on just the ID without the prefix?

Reply:
@mattcollier, yes, there are many capabilities associated with an instance.